app/app/controllers/cbv/base_controller.rb

```diff
@@ -71,7 +71,7 @@ def pinwheel
   end
 
   def agency_url
-    "https://www.nyc.gov/site/hra/help/snap-application-frequently-asked-questions.page"
+    site_config[@cbv_flow.site_id].agency_contact_website
```

Comment:
Let's do it a bit more safely:

```suggestion
    current_site&.agency_contact_website
```